src/components/common/ColonyActions/helpers/mapItemToMessageFormat.tsx

```diff
@@ -28,16 +28,17 @@ export const mapItemToExpectedFormat = (
   //   item.decimals,
   //   item.reputationChange,
   // );
+
   return {
     ...item,
     amount: (
       <Numeral
         value={item.amount ?? 0} // @TODO: getAmount(item.actionType, item.amount)
```

Comment:
Could you open an issue to track it? This is for getting the amount after network fees, right?

Reply:
Yes, left some more notes at the bottom of that file. https://github.com/JoinColony/colonyCDapp/issues/256